src/main/scala/monix/nio/WatchServiceObservable.scala

```diff
@@ -0,0 +1,75 @@
+package monix.nio
```

Comment:
If the service is used only for files we should move it into the file package. 
Also, please check if we can reuse some of the logic